database/youtube/web-development.json

```diff
@@ -61,5 +61,22 @@
         "category": "youtube",
         "subcategory": "web-development",
         "language": "hindi"
+    },
+    {
+        "name": "Bro Code",
+        "description": "This channel is provide a web development course for both beginner and pros.",
+        "url": "https://www.youtube.com/@BroCodez",
+        "category": "youtube",
+        "subcategory": "web-development",
+        "language": "English"
+    },
+    {
+        "name": "DesignCourse",
+        "description": "This channel is provide a we take you from zero to a fullstack developer.",
```

Comment:
Hello, again @darkweb907! 😃 There are some grammatical errors here. [Grammarly](https://www.grammarly.com/a?utm_source=google&utm_medium=cpc&utm_campaign=brand_core_usgroupb&utm_content=brandcoreusgroupb&utm_term=grammarly&matchtype=e&placement=&network=g&gclid=CjwKCAiAmJGgBhAZEiwA1JZoll50H1NdIkgXeHmG1mBtxCjt7uIMPN6IPIv8w4DQWMB32L5KCQAXpRoCr4cQAvD_BwE&gclsrc=aw.ds) has been helpful with my PRs so I highly recommend it, especially the free tier. Also, interesting source but consider adding more detail to it. What makes this video different from other YouTubers who teach fullstack development? I hope you find my feedback helpful! 😄 Good luck! 😄